Dockerfile

```diff
@@ -12,7 +12,7 @@ WORKDIR /app
 
 COPY . .
 
-RUN CYPRESS_INSTALL_BINARY=0 npm ci --no-audit --production && \
+RUN npm ci --no-audit --production && \
```

Comment:
don't have to worry about this as cypress no longer present in prod dependencies